c2rust-analyze/src/main.rs

```diff
@@ -424,7 +424,7 @@ fn run(tcx: TyCtxt) {
     for &ldid in &all_fn_ldids {
         let ldid_const = WithOptConstParam::unknown(ldid);
         let mir = tcx.mir_built(ldid_const);
-        let mir = mir.borrow();
+        let mir: std::cell::Ref<Body> = mir.borrow();
```

Comment:
What's this needed for?  Failed `rust-analyzer` intellisense?

Also,
```suggestion
        let mir: cell::Ref<Body> = mir.borrow();
```

Reply:
addressed in 0fb24480

Reply:
If it's helpful for intellisense, it's fine to keep it; just wondering.

Reply:
I think you had it right, it was a rust-analyzer thing and I did not add it intentionally